releng/org.eclipse.xtend.sdk.feature/feature.xml

```diff
@@ -32,7 +32,7 @@ http://www.eclipse.org/legal/epl-v10.html Description here.
       <import plugin="org.eclipse.xtend.lib.macro" version="2.17.0" match="equivalent"/>
       <import plugin="org.eclipse.xtend.lib.source" version="2.17.0" match="equivalent"/>
       <import plugin="org.eclipse.xtend.lib.macro.source" version="2.17.0" match="equivalent"/>
-      <import plugin="com.google.guava"/>
+      <import plugin="com.google.guava" version="21.0.0" match="greaterOrEqual"/>
```

Comment:
why greaterOrEqual here, not equivalent?

Reply:
copy & paste from others

Reply:
equivalent - dependent plug-in version must be at least at the version specified, or at a higher service level (major and minor version levels must equal the specified version).
compatible - dependent plug-in version must be at least at the version specified, or at a higher service level or minor level (major version level must equal the specified version).
greaterOrEqual - dependent plug-in version must be at least at the version specified, or at a higher service, minor or major level.

=> should io pick  compatible?

Reply:
updated to compatible